src/main/java/com/byznass/tiolktrack/resource/jaxrs/exception/mapper/NoGpsWIthIdExceptionMapper.java

```diff
@@ -0,0 +1,31 @@
+package com.byznass.tiolktrack.resource.jaxrs.exception.mapper;
+
+import com.byznass.tiolktrack.kernel.dao.NoGpsWithIdException;
+
+import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON_TYPE;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+
+@Provider
+public class NoGpsWIthIdExceptionMapper implements ExceptionMapper<NoGpsWithIdException> {
+
+	private static final String RESPONSE = "{\n\t\"Error message\": \"%s\"\n}";
+
+	@Inject
+	public NoGpsWIthIdExceptionMapper() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS1186)